docs/contributing/process.md

```diff
@@ -0,0 +1,15 @@
+# Enso Team Process
```

Comment:
Should we discern between IDE team and Enso Team? (this process seems to apply only to the IDE team)

What is the audience of this document? Since it describes our internal process (and references data available only through zenhub) I'd consider putting this in the internal repo.
Nothing secret here but I don't think this information is  useful to community.

Reply:
Michal, thi is meant for the IDE team now, maybe it will be applied to other teams in the future if it will work well here. Regarding moving it to internal repo - I this its worth keeping it here. As soon as we will have more contributors its worth including them in the process (at least those who are very active). Its also nice to be very transparent regarding our process - it just shows people how we are working and I always appreciate companies which are so transparent regarding their processes. Does it make sense to you? :)

Reply:
Actually, I meant to IDE team process. When it will be extended to the whole Enso, then we should change the title.

Reply:
@wdanilo Yes. I mostly wonder if this is the right place, since looking at path "contributing/process.md" it can suggest to the community that this is a process that needs to be followed to contribute. 
Though I'm not sure how it should be placed.